client_plugin/utils/refcount/refcnt.go

```diff
@@ -428,6 +430,16 @@ func (r *RefCountsMap) syncMountsWithRefCounters(d drivers.VolumeDriver) {
 	defer r.mtx.Unlock()
 
 	for vol, cnt := range r.refMap {
+		// If cnt is nil then this volume entry under mount root can be
+		// purged and the volume detached (if it is attached).
+		if cnt == nil {
```

Comment:
Yes, I didn't want to allocate mem. for the refCount structs for these entries and hence left them as empty entries. Can do it eitherway, allocate the structs and mark those as mounted = false and then remove them. Another thing is i'm using UnmountVolume() when clearly these volumes aren't mounted, I'll add a Detach() to the Drivers interface so the unmount error doesn't show up in the logs.